uisdk/src/main/java/com/karhoo/uisdk/screen/booking/map/BookingModeView.kt

```diff
@@ -41,6 +58,57 @@ class BookingModeView @JvmOverloads constructor(context: Context,
         }
     }
 
+    override fun displayDatePicker(minDate: Long, maxDate: Long, timeZone: String) {
```

Comment:
Could we create a static method for this and displayTimePicker() so that we modify also the AddressBarView.kt to implement them?